internal/models/models.go

```diff
@@ -578,6 +657,123 @@ func (siw *ServerInterfaceWrapper) GetApiV1TrustRootMetadataInfo(w http.Response
 	handler.ServeHTTP(w, r)
 }
 
+// GetApiV1TrustTarget operation middleware
+func (siw *ServerInterfaceWrapper) GetApiV1TrustTarget(w http.ResponseWriter, r *http.Request) {
+
+	var err error
+
+	// Parameter object where we will unmarshal all parameters from the context
+	var params GetApiV1TrustTargetParams
+
+	// ------------- Required query parameter "tufRepositoryUrl" -------------
+
+	if paramValue := r.URL.Query().Get("tufRepositoryUrl"); paramValue != "" {
+
+	} else {
```

Comment:
Is this auto-generated code? The empty if clause above looks a little weird, but I guess autogeneration could do that.